tests/integration/thrift/test_thriftrw.py

```diff
@@ -22,16 +22,12 @@
 
 import pytest
 
-from ...util import get_thrift_file
-
 from tchannel import TChannel
-from tchannel import thrift
 
 
 @pytest.fixture
-def keyvalue(tmpdir, request):
-    thrift_file = get_thrift_file(tmpdir)
-    return thrift.load(request.node.name, str(thrift_file))
+def keyvalue(thrift_module):
```

Comment:
this fixture is doing nothing right now. Shall we delete it and make other test just use thrift_module